oss_src/lambda/python_api.cpp

```diff
@@ -21,10 +21,34 @@ namespace lambda {
 
 namespace python = boost::python;
 namespace fs = boost::filesystem;
-/**
- * Enforce the sys path to be the same as the GLC client's sys path.
+
+/** Initialize python.  Does not release the GIL. 
  */
-void set_gl_sys_path() {
+void init_python(const std::string& root_path, bool debug_mode) { 
+
+  /** Define some info and error handling macros for this function to
+   *  handle the debug_mode diagnostic stuff.  This allows users to
+   *  run python _pylambda_worker.py and get a report of what's going
+   *  on and how far things work to. 
+   */
+  
+#define __INFO(...) do {                                        \
+    std::cout << "INFO: " << __VA_ARGS__ << std::endl;          \
+  } while(false)
+
+#define __ERROR(...) do {                                       \
+    std::cerr << "ERROR: " << __VA_ARGS__ << std::endl;         \
+    std::ostringstream ss; ss << __VA_ARGS__;                   \
+    throw(ss.str());                                            \
+  } while(false)
+  
```

Comment:
So why does this file in particular have novel info and error handling macros?
Why not just the logger like everything else?